tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "lib": ["DOM", "es2021"],
```

Comment:
Why is this needed?

Reply:
The lib option allows to add/specify types libraries in the project.  Here i had to add "es2021" because I use the newly es2021 `String.protype.replaceAll` feature.

And since I've added lib, i had an DOM type error (`parent.postMessage(JSON.stringify(messageData), '*')` so i've added too.